packages/components/src/components/button/button.tsx

```diff
@@ -198,19 +201,22 @@ export class Button {
         {this.styles && <style>{this.styles}</style>}
 
         {this.href ? (
-          <a
-            ref={(el) => (this.focusableElement = el)}
-            class={this.getCssClassMap()}
-            href={this.href}
-            download={this.download}
-            target={this.target}
-            rel={this.target === '_blank' ? 'noopener noreferrer' : undefined}
-            part={basePart}
-            tabIndex={this.innerTabindex}
-          >
-            <slot />
-          </a>
-        ) : (
+            <a
+              ref={(el) => (this.focusableElement = el)}
+              class={this.getCssClassMap()}
+              href={this.href}
+              download={this.download}
+              target={this.target}
+              rel={this.target === '_blank' ? 'noopener noreferrer' : undefined}
+              part={basePart}
+              tabIndex={this.innerTabindex}
+              role="link"
+              aria-disabled={this.disabled}
```

Comment:
This needs to be `aria-disabled={this.disabled ? 'true' : null}` so it actually render the "true" value, otherwise is not valid I think.

Reply:
true, removed in favor of your implementation